circle.yml

```diff
@@ -860,9 +869,20 @@ commands:
       - clone-repo-and-checkout-branch:
           repo: <<parameters.repo>>
           pull_request_id: <<parameters.pull_request_id>>
+      - run:
```

Comment:
As I was debugging the various failures in the recipes/kitchensink projects, I noticed that we were throwing a bunch of 'bad engine' warnings due to our usage of the monorepo's .node-version while installing/running these other projects. So I added a quick curl here for the script in the monorepo and source it here to ensure we are installing the projects node-version, if it has one specified. I don't think this was the root cause of the issues I was seeing, but it might head off some weirdness in the future.